.github/workflows/aks-letsencrypt.yml

```diff
@@ -23,18 +23,10 @@ on:
     - cron:  '0 3 * * *'
   workflow_dispatch:
     inputs:
-      azure_credentials:
-        description: "AZURE_CREDENTIALS"
-        required: false
-        default: ""
       aks_domain:
         description: "AKS_DOMAIN to use, managed via Route53's AWS_ZONE_ID"
         required: false
         default: ""
-      aws_zone_id:
-        description: "AWS_ZONE_ID"
-        required: false
-        default: ""
       keep_cluster:
```

Comment:
I suspect that Enrico talked about `keep_cluster` ?

Reply:
Exactly :smile:

Reply:
Ahh, I was confused by my own first thoughts on 'aws_zone_id' - which are true.
So the short answer is: I'll make 'keep_cluster' type 'choice' as well defaulting to 'No'.
About 'aws_zone_id': I removed it, because I remember someone considered it as "secret" as well. But AFAIK one can't make any use of 'aws_zone_id' without being authenticated - so I could bring it back.
Finally because I realized that the current state of having '*_domain' as dispatch input, without 'aws_zone_id' doesn't make sense.
So I should bring back 'aws_zone_id', or remove '*_domain' as well - what do you think?

Reply:
Made keep_cluster type choice, and removed domain input completely.